tools/EventClients/Clients/OSXRemote/XBMCHelper.m

```diff
@@ -253,6 +253,8 @@ - (NSString *)buttonNameForButtonCode:(HIDRemoteButtonCode)buttonCode
       break;
 		case kHIDRemoteButtonCodeMenuHold:
 			return (@"Menu (hold)");
+      break;
+        default:
```

Comment:
Why not mention the missing cases instead of default?
Last time I looked at it, there were cases which should be handled by this switch statement.

Reply:
I don't know objective-C and have never used this helper. Can you take care of this?

Reply:
@Memphiz or @FernetMenta could you add the 8 missing labels?

Reply:
`kHIDRemoteButtonCodePlay` and `kHIDRemoteButtonCodePlayPause` should be handled. For the other cases you can leave the default case.

Reply:
I added `kHIDRemoteButtonCodePlay`, but it seems `kHIDRemoteButtonCodePlayPause` is already handled?

Reply:
my fault mixed something up, correct ones are `kHIDRemoteButtonCodePlay` and `kHIDRemoteButtonCodePlayHold`